praw/models/reddit/subreddit.py

```diff
@@ -198,6 +199,13 @@ def moderator(self):
         return self._moderator
 
     @property
+    def modmail(self):
+        """An instance of :class:`.Modmail`."""
```

Comment:
I've encounterd same problem when introduced LiveUpdate class (b9f04d26e4aceffdfdeeafab60db6a8d4531a361). In this case, adding `docs/code_overview/models/modmail.rst` and appropriate entry to the `praw_models.rst` may solve the problem.